### PR TITLE
Chat command improvements

### PR DESCRIPTION
**Pull request by zrisher**
Partial implementation of #14

### CODE REVIEW

File: Data/Scripts/GardenConquest/Core/CommandProcessor.cs [hunk 1]
```diff
@@ -21,20 +21,72 @@ class CommandProcessor {
...
 			"/gc help - Show this screen\n" +
-			"/gc help [category] - Show help for a category.  Categories are:\n" +
-			"    limits - Information on grid class limitations\n";
+			"/gc help [category] - Show help for a category.  Categories:\n" +
+			"        classes     - Ship Classes and their limits\n" +
+			"        classifiers - Hull Classifier blocks\n" +
+			"        cps            - Control Points\n" +
+			"        licenses    - Ship License components\n" +
+			"/gc fleet - Information on your faction's fleet \n" +
+			"/gc fleet remove \"Ship Name\"- Disown a ship";
```
Comment (stackcollision) [inline Data/Scripts/GardenConquest/Core/CommandProcessor.cs:37]: Not sure if this is a great idea.  Depends what you mean by "disown"

File: Data/Scripts/GardenConquest/Core/CommandProcessor.cs [hunk 2]
```diff
@@ -21,20 +21,72 @@ class CommandProcessor {
...
+			"by a player or disappears entirely.\n\n" + 
+			"For more info on Classes, type \"/gc help classes\"";
+
+		private static string s_HelpCPsText1 =
+			"Control Points are areas of the map you can hold and control. " +
+			"Each CP has a Radius, Reward, and Round Time. On this server, " +
+			"these are:\n\n";
+
+		private static string s_HelpCPsText2 =
+			"\n\nIn order to win a 15 minute round, you or your faction must " +
```
Comment (stackcollision) [inline Data/Scripts/GardenConquest/Core/CommandProcessor.cs:64]: No direct mentions of times.  It's configurable

File: Data/Scripts/GardenConquest/Core/CommandProcessor.cs (hunk 1 above)
Comment (zrisher) [inline Data/Scripts/GardenConquest/Core/CommandProcessor.cs:37]: I.e. you spawn in your Unlicensed ship. It gets wrecked. You spawn in another unlicensed spawn ship, but enough of your original ship remains to get picked up as counting towards your Unlicensed limit, so your new ship is going to derelict in 2 hours. You use this command to force a dereliction on your old ship instead.

File: Data/Scripts/GardenConquest/Core/CommandProcessor.cs (hunk 2 above)
Comment (zrisher) [inline Data/Scripts/GardenConquest/Core/CommandProcessor.cs:64]: Would you mind including for now and then I can adjust once I see how you fill out the info for the other configuration options?

File: Data/Scripts/GardenConquest/Core/CommandProcessor.cs (hunk 1 above)
Comment (zrisher) [inline Data/Scripts/GardenConquest/Core/CommandProcessor.cs:37]: Probably an even more common use case would be when your ship gets wrecked in battle, is flying off at a million m/s and no one catches it to get your hull classifier, but it's still counting against your limits.

File: Data/Scripts/GardenConquest/Core/CommandProcessor.cs (hunk 2 above)
Comment (stackcollision) [inline Data/Scripts/GardenConquest/Core/CommandProcessor.cs:64]: That's fine

**Pull request merged**